src/EprPrnIntegration.Common/Service/EmailService.cs

```diff
@@ -153,21 +144,16 @@ public void SendUpdatedPrnsReconciliationEmailToNpwd(DateTime reportDate, string
 
     public void SendUpdatedOrganisationsReconciliationEmailToNpwd(DateTime reportDate, int reportDataRowsCount, string reportCsv)
     {
+        ArgumentException.ThrowIfNullOrWhiteSpace(reportCsv, nameof(reportCsv));
+
         var templateId = _messagingConfig.NpwdReconcileUpdatedOrganisationsTemplateId;
         var emailAddress = _messagingConfig.NpwdEmail;
-        var filename = $"updatedorganisations_{reportDate:yyyyMMdd}.csv";
 
         var messagePersonalisation = new Dictionary<string, object>
         {
-            {
-                "UpdatedDate", reportDate.ToString("dd/MM/yyyy")
-            },
-            {
-                "RowCount", reportDataRowsCount
-            },
-            {
-                "link_to_file", NotificationClient.PrepareUpload(System.Text.Encoding.UTF8.GetBytes(reportCsv), filename)
-            }
+            ["UpdatedDate"] = reportDate.ToString("dd/MM/yyyy"),
+            ["RowCount"] = reportDataRowsCount,
+            ["link_to_file"] = reportCsv          
```

Comment:
Since now we are doing changes on Template as well, Can we change link_to_file to something like csvData inline with what we are doing.